<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>599</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="background" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>811</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#background {
background-color: #178fd6;

}

</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>380</x>
       <y>100</y>
       <width>61</width>
       <height>51</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap>../../../../Desktop/MOVIE TICKETING SYSTEM/img/signuptxt.png</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QWidget" name="rect" native="true">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>190</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#rect{
background-color: white;
border: 1px white;
border-radius: 20px;
}</string>
     </property>
     <widget class="QLabel" name="label_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../Desktop/MOVIE TICKETING SYSTEM/img/username.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="user">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>5</y>
        <width>221</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#user{
border: TRANSPARENT;
}</string>
      </property>
      <property name="placeholderText">
       <string>Username</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="rect_2" native="true">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>250</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#rect_2{
background-color: white;
border: 1px white;
border-radius: 20px;
}</string>
     </property>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../Desktop/MOVIE TICKETING SYSTEM/img/password.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="admin_pass">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>5</y>
        <width>221</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#admin_pass{
border: TRANSPARENT;
}</string>
      </property>
      <property name="placeholderText">
       <string>Password</string>
      </property>
     </widget>
    </widget>
    <widget class="QPushButton" name="enter">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>370</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">#enter{
background-color: #034488;
color: white;
	font: 11pt &quot;MS Shell Dlg 2&quot;;
border: 1px solid white;
border-radius: 20px;
}</string>
     </property>
     <property name="text">
      <string>Create An Account</string>
     </property>
    </widget>
    <widget class="QPushButton" name="signUpButton">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>430</y>
       <width>291</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
background-color: white;
text-size: 10px;
border-radius: 10px;
}

QPushButton:hover{
	color:rgb(73, 147, 221);
	background-color: rgb(13, 27, 38);
}
#signUpButton {
background-color: TRANSPARENT;
}
#signUpButton:hover{
color: rgb(0, 0, 255);
}</string>
     </property>
     <property name="text">
      <string>or login</string>
     </property>
    </widget>
    <widget class="QWidget" name="rect4" native="true">
     <property name="geometry">
      <rect>
       <x>260</x>
       <y>300</y>
       <width>291</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">#rect4{
background-color: white;
border: 1px white;
border-radius: 20px;
}</string>
     </property>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>5</y>
        <width>41</width>
        <height>31</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="pixmap">
       <pixmap>../../../../Desktop/MOVIE TICKETING SYSTEM/img/password.png</pixmap>
      </property>
      <property name="scaledContents">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QLineEdit" name="admin_pass1">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>5</y>
        <width>221</width>
        <height>30</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">#admin_pass1{
border: TRANSPARENT;
}</string>
      </property>
      <property name="placeholderText">
       <string>Confirm Password</string>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QWidget" name="mainBG" native="true">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>0</y>
      <width>651</width>
      <height>611</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#mainBG {
background-color: #fffcf6;

}

</string>
    </property>
   </widget>
   <zorder>mainBG</zorder>
   <zorder>background</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
